galaxy_importer/main.py

```diff
@@ -16,37 +16,62 @@
 # along with Galaxy.  If not, see <http://www.apache.org/licenses/>.
 
 import argparse
+import json
 import logging
+from pprint import pprint
 import sys
 
 from . import collection
 
 
-def start_import(filepath):
-    """Outputs result of galaxy_importer import process.
+def main(args=None):
+    logging.basicConfig(
+        stream=sys.stdout,
+        format='%(levelname)s: %(message)s',
+        level=logging.INFO)
+    args = parse_args(args)
 
-    :param filepath: Artifact file to import.
-    """
-    json_data = collection.import_collection(filepath)
-    # TODO: make default output go to file
-    print(json_data)
+    json_data = call_importer(filepath=args.filepath)
+
+    if args.print_result:
+        pprint(json.loads(json_data))
+
+    write_output_file(json_data)
 
 
 def parse_args(args):
     parser = argparse.ArgumentParser(
         description='Import collection and run linting')
     parser.add_argument(
-        '-f', '--filepath', default=None, help='Artifact to import.')
+        '-f', '--filepath',
```

Comment:
Minor \ Naming: IMO `filepath` is too long, usually `file` or `input` or `artifact` is enough.

Also, if artifact file is a required argument for importer, usually it's better to make it a positional argument.

Reply:
Updated